specification/storage/resource-manager/Microsoft.Storage/stable/2021-08-01/storage.json

```diff
@@ -0,0 +1,5292 @@
+        },
+        "forestName": {
+          "type": "string",
+          "description": "Specifies the Active Directory forest to get."
+        },
+        "domainGuid": {
+          "type": "string",
+          "description": "Specifies the domain GUID."
+        },
+        "domainSid": {
+          "type": "string",
+          "description": "Specifies the security identifier (SID)."
+        },
+        "azureStorageSid": {
+          "type": "string",
+          "description": "Specifies the security identifier (SID) for Azure Storage."
+        },
+        "samAccountName": {
+          "type": "string",
+          "description": "Specifies the Active Directory SAMAccountName for Azure Storage."
+        },
+        "accountType": {
+          "type": "string",
+          "description": "Specifies the Active Directory account type for Azure Storage.",
+          "enum": [
+            "User",
+            "Computer"
+          ]
+        }
+      },
+      "required": [
+        "domainName",
+        "netBiosDomainName",
+        "forestName",
+        "domainGuid",
+        "domainSid",
+        "azureStorageSid"
+      ],
+      "description": "Settings properties for Active Directory (AD)."
+    },
+    "StorageAccountPropertiesCreateParameters": {
+      "properties": {
+        "allowedCopyScope": {
+          "type": "string",
+          "description": "Restrict copy to and from Storage Accounts within an AAD tenant or with Private Links to the same VNet.",
+          "enum": [
+            "PrivateLink",
+            "AAD"
+          ],
+          "x-ms-enum": {
+            "name": "AllowedCopyScope",
+            "modelAsString": true
+          }
+        },
+        "publicNetworkAccess": {
+          "type": "string",
+          "description": "Allow or disallow public network access to Storage Account. Value is optional but if passed in, must be 'Enabled' or 'Disabled'.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "PublicNetworkAccess",
+            "modelAsString": true
+          }
+        },
+        "sasPolicy": {
+          "$ref": "#/definitions/SasPolicy",
+          "description": "SasPolicy assigned to the storage account."
+        },
+        "keyPolicy": {
+          "$ref": "#/definitions/KeyPolicy",
+          "description": "KeyPolicy assigned to the storage account."
+        },
+        "customDomain": {
+          "$ref": "#/definitions/CustomDomain",
+          "description": "User domain assigned to the storage account. Name is the CNAME source. Only one custom domain is supported per storage account at this time. To clear the existing custom domain, use an empty string for the custom domain name property."
+        },
+        "encryption": {
+          "$ref": "#/definitions/Encryption",
+          "description": "Not applicable. Azure Storage encryption is enabled for all storage accounts and cannot be disabled."
```

Comment:
this description doesn't seem valid given CMK, etc... resides in this object

Reply:
Will update this.